saopaulo2024-2-edicao.html

```diff
@@ -0,0 +1,603 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Rails Girls São Paulo - 2024 - 2º Edição</title>
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <meta content="rails, railsgirls, rails girls, ruby on rails, women in tech, mulheres na computação" name="keywords">
+  <meta content="Rails Girls chegou em São Paulo e agora não sai mais! Venha para se divertir e aprender Ruby on Rails conosco!" name="description">
+
+  <!-- Favicons -->
+  <link href="img/favicon.png" rel="icon">
+  <link href="img/apple-touch-icon.png" rel="apple-touch-icon">
+
+  <!-- Google Fonts -->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,700,700i|Raleway:300,400,500,700,800" rel="stylesheet">
+
+  <!-- Bootstrap CSS File -->
+  <link href="lib/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Libraries CSS Files -->
+  <link href="lib/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+  <link href="lib/animate/animate.min.css" rel="stylesheet">
+  <link href="lib/venobox/venobox.css" rel="stylesheet">
+  <link href="lib/owlcarousel/assets/owl.carousel.min.css" rel="stylesheet">
+
+  <!-- Main Stylesheet File -->
+  <link href="css/style.css" rel="stylesheet">
+
+  <!-- =======================================================
+    Theme Name: TheEvent
+    Theme URL: https://bootstrapmade.com/theevent-conference-event-bootstrap-template/
+    Author: BootstrapMade.com
+    License: https://bootstrapmade.com/license/
+  ======================================================= -->
+</head>
+
+<body>
+
+  <!--==========================
+    Header
+  ============================-->
+  <header id="header">
+    <div class="container">
+
+      <div id="logo" class="pull-left">
+        <a href="#intro" class="scrollto"><img src="img/logo.png" alt="" title=""></a>
+      </div>
+
+      <nav id="nav-menu-container">
+        <ul class="nav-menu">
+          <li class="menu-active"><a href="#intro">Home</a></li>
+          <li><a href="#about">Sobre</a></li>
+          <li><a href="#schedule">Cronograma</a></li>
+          <li><a href="#faq">FAQ</a></li>
+          <li><a href="#supporters">Patrocinadores</a></li>
+          <li><a href="#venue">Local</a></li>
+          <li><a href="#speakers">Organizadoras</a></li>
+          <li><a href="#gallery">Fotos</a></li>
+        </ul>
+      </nav><!-- #nav-menu-container -->
+    </div>
+  </header><!-- #header -->
+
+  <!--==========================
+    Intro Section
+  ============================-->
+  <section id="intro" class="dark">
+    <div class="intro-container wow fadeIn">
+      <h1 class="mb-4 pb-0">Rails Girls São Paulo 2024<br><p class="mb-4 pb-0">Segunda Edição</p></h1>

```

Comment:
Pra descrição casar com o media kit, acho que poderia ser:
```suggestion
      <h1 class="mb-4 pb-0">Rails Girls São Paulo 2024<br><p class="mb-4 pb-0">Edição Imersão</p></h1>

```